term_bsd.go

```diff
@@ -7,7 +7,7 @@ import "syscall"
 type attr syscall.Termios
 
 func (a *attr) setSpeed(baud int) error {
-	var rate uint64
+	var rate uint32
```

Comment:
Why did you change this ? If you need to make a change for only freebsd, you'll need to copy this file into term_freebsd.go and alter the build tags on the other files.